metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/datahub_listener.py

```diff
@@ -629,11 +630,18 @@ def on_dag_start(self, dag_run: "DagRun") -> None:
             )
             self.emitter.emit(event)
 
+        browsePaths: List[BrowsePathEntryClass] = []
+        if self.config.platform_instance:
+            urn = make_dataplatform_instance_urn(
```

Comment:
Additionally to adding platform instance to the URN (which solves identity) and the browse paths (which solves the browsing experience in the navigation left-side panel in the UI), if platform instance the `dataPlatformInstance` aspect (which supports the platform instance filter in the main page of the search in the UI) should be emitted too.

Note that `dataPlatformInstance` is supported by both `DataJob` 
https://github.com/datahub-project/datahub/blob/7618af549c6386ded8b4405092d3e64999467a20/metadata-models/src/main/resources/entity-registry.yml#L74

and `DataFlow` entities
https://github.com/datahub-project/datahub/blob/7618af549c6386ded8b4405092d3e64999467a20/metadata-models/src/main/resources/entity-registry.yml#L98